app/cdap/components/Replicator/Create/Content/SelectColumnsWithTransforms/index.tsx

```diff
@@ -272,19 +298,17 @@ const SelectColumnsView = (props: ISelectColumnsProps) => {
           label={`${props.tableInfo.table} - Mappings, assessments and transformations`}
         />
         <ActionButtons>
-          {/* 
-            Will need to add this back for the 6.6 version https://cdap.atlassian.net/browse/CDAP-18593
-          <ButtonWithMarginRight variant="text" color="primary" onClick={props.toggle}>
+          <Button variant="text" color="primary" onClick={props.toggle}>
```

Comment:
Nit: can you use the pre-defined buttons from `shared/Buttons`? If you need to add a new one, please do so.